### DISCUSSION TITLE
Github Copilot theme used in the docs

### DISCUSSION THREAD

**Discussion by jvzaniolo** [community/community#13932] [Copilot Conversations]
Hey, I loved the theme you used in the [Github Copilot docs](https://copilot.github.com/). Any way you could share both dark/light themes with us?

Thanks!

| **Dark** | **Light** |
|----------|----------|
|![CleanShot 2022-03-30 at 6 51 45](https://user-images.githubusercontent.com/54036572/160937143-ef7bb2c2-3e1a-490c-9b38-7193505ab57b.png)|![CleanShot 2022-03-30 at 6 52 46](https://user-images.githubusercontent.com/54036572/160937214-345e8ed2-58a6-4cef-b921-ef77b1583c1d.png)|

**Comment by adoolaard** [comment]
The dark theme is freely available on VS Code.
https://marketplace.visualstudio.com/items?itemName=BenjaminBenais.copilot-theme

**Reply by jvzaniolo** [reply]
That’s not the official version used in the docs, is just something that someone created based on the official theme
